core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -124,6 +124,11 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         _;
     }
 
+    modifier noPendingWithdrawal() {
```

Comment:
I added this modifier because the same check is done in 6 functions. by pulling this to a separate modifier the gas used drops by ~ 500 bytes.